xios_examples/gen_netcdf.py

```diff
@@ -0,0 +1,290 @@
+import sys
+import os
+import argparse
+import netCDF4 as nc
+import numpy as np
+from .dataFunc import dataFunc
+
+# Global defaults here as needed for command line arguments defaults and argument defaults to run function
+defaults = {
+            'func_str': 'sinusiod',
+            'mesh_file': None,
+            'mesh_varname': None,
+            'nlat': 101,
+            'nlon': 100,
+            'nlatr': 81,
+            'nlonr': 80
+           }
+
+def create_ncfile(ncfile, nlat, nlon, func, dim_prefix='', dim_suffix='', data_prefix='', data_suffix=''):
+
```

Comment:
please may we  have a triple-quote docstring for this function?

Reply:
done